web/src/features/charts/BreakdownChart.tsx

```diff
@@ -123,7 +126,8 @@ function BreakdownChart({
             }}
             title={t('data-sources.title')}
             className="text-md"
-            isCollapsedAtom={dataSourcesCollapsedBreakdown}
+            isCollapsed={dataSourcesCollapsedBreakdown}
+            setState={setDataSourcesCollapsedBreakdown}
```

Comment:
The fact that these always end up the same length is very satisfying.

Reply:
![length](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExc2p5eHExODJqNXVtbWdmY3Z2bG9xZG9kcnBoeXl1dWprMDk3OHEzMiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/4WFi6XibeDTM51XHQi/giphy.gif)